### PR TITLE
chore(common): pass ldml import path from caller 🙀 

### PR DESCRIPTION
**Pull request by srl295**
- remove use of import.meta.url in libraries
- use relative path between tests
- may need some improvement before it works via npm!

Fixes: #10358

@keymanapp-test-bot skip

### CODE REVIEW

File: developer/src/kmc/src/commands/buildClasses/BuildLdmlKeyboard.ts [hunk 1]
```diff
@@ -1,10 +1,14 @@
 import * as kmcLdml from '@keymanapp/kmc-ldml';
 import { KeymanFileTypes } from '@keymanapp/common-types';
 import { CompilerOptions, CompilerCallbacks } from '@keymanapp/developer-utils';
-import { LDMLKeyboardXMLSourceFileReader } from '@keymanapp/developer-utils';
 import { BuildActivity } from './BuildActivity.js';
 import { fileURLToPath } from 'url';
 
+function defaultImportsURL(): [string,string] {
+  // TODO: Need to pull from node_modules!
+  return ['../../../../../common/web/utils/build/src/import/', import.meta.url];
```
Comment (srl295) [inline developer/src/kmc/src/commands/buildClasses/BuildLdmlKeyboard.ts:9]: @mcdurdin not sure this will fly when installed from NPM, what do you think?

Comment (mcdurdin) [reply developer/src/kmc/src/commands/buildClasses/BuildLdmlKeyboard.ts:9]: No, it won't fly there or in Keyman Developer bundled install.

**Comment by mcdurdin** [comment]
Now I wonder if we need to cross this bridge at present -- we are using this only in a Node environment now, and so it should just work. If we do move to web front end compiler tooling at some point, we can cross this data file bridge then. 

We understand the problem but it is non-trivial busy-work to resolve -- data file deployment, retrieval etc. Each of the solutions has corresponding downsides. So I think we can close this PR and also the corresponding issue.

**Comment by srl295** [comment]
> Now I wonder if we need to cross this bridge at present -- we are using this only in a Node environment now, and so it should just work. If we do move to web front end compiler tooling at some point, we can cross this data file bridge then. 
> 
> 
> 
> We understand the problem but it is non-trivial busy-work to resolve -- data file deployment, retrieval etc. Each of the solutions has corresponding downsides. So I think we can close this PR and also the corresponding issue.

It's conceivable that we'l want this to work for the visual editor to be able to compile in-browser.  That said, it's probably reasonable to close issue+pr, can always dust it off later as needed.

**Pull request closed without merging**